File: algorithm/C++/0215-Kth_Largest_Element_in_an_Array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int, vector<int>, less<int>> myPriortyQueue(less<int>(), nums);
        for (int i=0; i<k-1; ++i) {
            myPriortyQueue.pop();
        }
        return myPriortyQueue.top();
    }
};

File: algorithm/Java/0215-Kth_Largest_Element_in_an_Array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> myHeap = new PriorityQueue<Integer>((n1, n2) -> n2 - n1);
        for (int i=0; i<nums.length; ++i) {
            myHeap.add(nums[i]);
        }
        for (int i=0; i<k - 1; ++i) {
            myHeap.poll();
        }
        return myHeap.poll();
    }
}
